daemon/routes/ifgroup/ifgroup.go

```diff
@@ -0,0 +1,118 @@
+package ifgroup
+
+import (
+	"errors"
+	"fmt"
+	"net"
+	"sync"
+
+	"github.com/vishvananda/netlink"
+)
+
+// Group is a group to be used within the application while it works
+const Group = 0xe1f1
+
+var (
+	// ErrAlreadySet defines that NetlinkManager is already set.
+	ErrAlreadySet = errors.New("already set")
+	// ErrNotSet defines that NetlinkManager is not set yet.
+	ErrNotSet = errors.New("not set")
+)
+
+// Manager is responsible for manager ifgroup to the network interfaces
+type Manager interface {
+	Set() error
+	Unset() error
+}
+
+// NetlinkManager is responsible for setting ifgroup to all of the network interfaces given
+// by a deviceList function and reverting the changes when needed.
+type NetlinkManager struct {
+	group      int
+	deviceList func() ([]net.Interface, error)
+	set        bool
+	backup     map[int]uint32
+	mu         sync.Mutex
+}
+
+// NewNetlinkManager is a default constructor for NetlinkManager
+func NewNetlinkManager(deviceList func() ([]net.Interface, error)) *NetlinkManager {
```

Comment:
Will there be uts implemented for NetlinkManager later on? If not, maybe `ListPhysical` could be just called directly?

Reply:
Integration `Link` test is implemented. It still uses `ListPhysical` but any other function could be used